streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -96,13 +94,7 @@
 import static org.apache.kafka.streams.integration.utils.IntegrationTestUtils.safeUniqueTestName;
 import static org.apache.kafka.streams.integration.utils.IntegrationTestUtils.waitForApplicationState;
 
-import static org.easymock.EasyMock.anyBoolean;
 import static org.apache.kafka.test.TestUtils.waitForCondition;
-import static org.easymock.EasyMock.anyInt;
```

Comment:
Is `easymock` still used by streams module? If not, could you remove it from `build.gradle`?

Reply:
I am worried that the current changes for `build,gradle` will cause conflicts. I am going to prepare a separate PR after all related issues are merged.